components/offscreen/offscreen.js

```diff
@@ -1,7 +1,10 @@
 import { css, html, LitElement } from 'lit';
 import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
 
-const offscreenStyleDeclarations = css`
+/**
```

Comment:
I don't love exporting these like this, and I considered moving them into a generator method, like we've done for [typography styles](https://github.com/BrightspaceUI/core/blob/main/components/typography/styles.js#L245). It would be more flexible, but we need these applied to the dropdown content host element, and our CSS selector Regex validation used by those generated methods doesn't allow `:host` [for this reason](https://github.com/BrightspaceUI/core/pull/2698#discussion_r931527209). I could be convinced to go that route, but it opens up the possibility for more invalid scenarios, so I decided to take the simpler approach here, given that we keep these under wraps either way.